shared/utils/instruction-parsing.hpp

```diff
@@ -1,7 +1,6 @@
 #include <array>
```

Comment:
I'm just leaving this comment at the top for instruction-parsing, but this is something that would optimally be marginally resolved in terms of functionality.